web/app/components/MobileNavigationItem/MobileNavigationItem.tsx

```diff
@@ -0,0 +1,75 @@
+import { Icon, Icons, Link } from '~/components';
+import { useEffect, useState } from 'react';
+import { Transition } from '@headlessui/react';
+import classNames from 'classnames';
+import { useLocation } from 'remix';
+
+enum Variant {
+  ScrollToTop,
+  Home,
+}
+
+export interface MobileNavigationItemProps {
+  variant: Variant;
+}
+
+export function MobileNavigationItem({ variant }: MobileNavigationItemProps) {
+  const [visible, setvisible] = useState<boolean>(false);
```

Comment:
Isn't necessary pass type in this useState, as you pass a value, TS will infer a type as Boolean.